DOC: add development doc (#189)
xprobe-inc/xorbits: doc/source/getting_started/deployment.rst

Deployment
~~~~~~~~~~

Xorbits can be deployed on many platforms, see below for what you need.

========================================= ============================================================
Deployment                                Description
========================================= ============================================================

> :ref:`Local <deployment_local>`           Running Xorbits on a local machine, e.g. laptop

Reviewer: ```suggestion
:ref:`Local <deployment_local>`           Run Xorbits on a local machine, e.g.  your laptop
```

:ref:`Cluster <deployment_cluster>`       Deploy Xorbits to existing cluster via command lines

> :ref:`Kubernetes <deployment_kubernetes>` Deploy Xorbits to existing k8s cluster via python codes

Reviewer: ```suggestion
:ref:`Kubernetes <deployment_kubernetes>` Deploy Xorbits to existing k8s cluster via python code
```

> :ref:`Cloud <deployment_cloud>`           Deploy Xorbits to various cloud platforms via python codes

Reviewer: ```suggestion
:ref:`Cloud <deployment_cloud>`           Deploy Xorbits to various cloud platforms via python code
```
